scalameta/parsers/shared/src/main/scala/scala/meta/internal/parsers/ScalametaParser.scala

```diff
@@ -226,6 +226,10 @@ class ScalametaParser(input: Input)(implicit dialect: Dialect) { parser =>
     accept[LeftParen]
     inParensAfterOpen(body)
   }
+  @inline final def inParensOr[T](body: => T)(alt: => T): T = {
+    accept[LeftParen]
+    inParensAfterOpenOr(body)(alt)
```

Comment:
```suggestion
    if (acceptOpt[LeftParen]) inParensAfterOpenOr(body)(alt)
    else alt
```
?
Otherwise, the method seems misleading since it doesn't accept no parens.

Reply:
for both comments: the method requires _parens_ but it doesn't require anything _in_ them (that is, allows the closing paren right away).

Reply:
Maybe then `inParens` instead `inParensOr` : 
```
   @inline final def inParens[T](body: => T)(whenEmpty: => T): T = {
```

Reply:
there is already a method called that way, for cases when there's no "or", meaning that an immediate closing paren is not expected... let me review.

Reply:
so, here's what we have:

- both parameters (_body_ and _alt_) are of the same type, so we can't reorder them to disambiguate from regular `inParens`
- there are several methods named similarly (two sets right below) and behaving similarly.

Reply:
thanks, will rename the `alt` parameter.

Reply:
added a separate PR for that, #2967.

Reply:
@tgodzik that PR has been merged. please let me know if there are any other adjustments to make.